explorer/cnquery_explorer.proto

```diff
@@ -136,6 +136,7 @@ enum Action {
   DEACTIVATE = 2;
   ACTIVATE = 3;
   IGNORE = 4;
+  OUT_OF_SCOPE = 5;
```

Comment:
what is the difference between this and ignore?

Reply:
Out of scope is closer to `DEACTIVATE` rather than `IGNORE`. Essentially, out of scope will create a different type of exception that doesn't show up in reporting or anywhere else but it will still disable execution